packages/@vivliostyle/theme-base/css/partial/toc.css

```
@@ -2,9 +2,9 @@
   --vs-toc--anchor-align-items: baseline;
   --vs-toc--anchor-text-decoration: none;
   --vs-toc--anchor-text-decoration-on-hover: underline;
-  --vs-toc--anchor-text-indent: 1ch;
   --vs-toc--marker-content: counter(vs-counter-toc) ' ';
   --vs-toc--marker-display: none;
+  --vs-toc--marker-padding-inline-end: 1ch;
   --vs-toc--marker-text-align: start;
```

- `--vs-toc--marker-content` は `counter()` でなく `counters()` を使うように直した方がよいでしょう。
- 6レベルまでの番号を表示する幅を確保したのに、`--vs-toc--marker-text-align` の値が `start` だと、その幅の中で左寄せになって、番号のあとに大きなアキができてしまいます。

```suggestion
  --vs-toc--marker-content: counters(vs-counter-toc, '.') ' ';
  --vs-toc--marker-display: none;
  --vs-toc--marker-padding-inline-end: 1ch;
  --vs-toc--marker-text-align: end;
```